packages/algob/test/mocks/tx.ts

```diff
@@ -38,3 +38,13 @@ export const mockAssetInfo: AssetInfo = {
     clawback: undefined
   }
 };
+
+const mockProgram = new Uint8Array([
+  2, 32, 4, 1, 4, 100, 144, 78, 49, 16,
+  34, 18, 49, 16, 35, 18, 17, 49, 8, 36,
+  14, 16, 49, 18, 36, 14, 16, 49, 32, 50,
+  3, 18, 16, 49, 9, 50, 3, 18, 16, 49,
+  1, 37, 14, 16
+]);
```

Comment:
Do we need to populate this? Could `mockProgram = new Uint8Array(0);` work?

Reply:
no it doesn't